table.go

```diff
@@ -481,6 +481,8 @@ func (t *Table) Iterator(
 		return err
 	}
 
+	callback := callbacks[0]
```

Comment:
Can we add that this uses index 0 because we don't support concurrency currently?